PWGHF/TableProducer/candidateCreatorToXiPi.cxx

```diff
@@ -77,12 +80,18 @@ struct HfCandidateCreatorToXiPi {
 
   // filter to use only HF selected collisions
   Filter filterSelectCollisions = (aod::hf_sel_collision::whyRejectColl == 0);
+  Filter filterSelectTrackIds = (aod::hf_sel_track::isSelProng >= 4);
 
   using SelectedCollisions = soa::Filtered<soa::Join<aod::Collisions, aod::HfSelCollision>>;
-  using MyTracks = soa::Join<aod::BigTracks, aod::TracksDCA, aod::TrackSelection>;
+  using MyTracks = soa::Join<aod::BigTracks, aod::TracksDCA, aod::HfPvRefitTrack>;
+  using FilteredHfTrackAssocSel = soa::Filtered<soa::Join<aod::TrackAssoc, aod::HfSelTrack>>; // HfTrackAssoc
```

Comment:
Move `using` before filters.

Reply:
Ok

Reply:
Maybe there was a misunderstanding. All the `using` declarations should be before all the filters. (See the HF guidelines.)

Reply:
Even if a filter is then to be used inside the using?

Reply:
That cannot happen. Filters are not used anywhere in the code once they are declared.

Reply:
Ok